Vagrantfile

```diff
@@ -126,6 +126,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # Start an OpenShift cluster
     # Currently this only works with the (default) VirtualBox provider.
 
+    # Tag configuration as stale when provisioning a dev cluster to
+    # ensure that nodes can wait for fresh configuration to be generated.
+    if ARGV[0] =~ /^up|provision$/i and not ARGV.include?("--no-provision")
+      system('test -d ./openshift.local.config && touch ./openshift.local.config/.stale')
```

Comment:
That's the point of adding 'rm -rf ${config_root}/openshift.local.*' on line 85 of provision-util.sh - to clear stale configuration before the master generates new configuration.  The nodes would hang forever if the configuration were not cleared.